### ISSUE TITLE
mage test:integration hangs (release 1.11)

### ISSUE THREAD

**Issue by rleighton** [panther-labs/panther#1865]
### Describe the bug

`mage test:integration` never finishes.

Odd to see this since this should be part of the every release test. Maybe something in my env?

NOTE: I hit this on master doing tests before posting a PR. 

### Steps to reproduce

- checkout release-1.11
- `mage deploy`
- `mage test:integration`

### Expected behavior

Finishes.

### Environment

- release-1.11.0

### Screenshots

![image](https://user-images.githubusercontent.com/18419762/97113369-4b6e8c00-16c0-11eb-8f7b-62288f0fa029.png)

**Comment by austinbyers** [comment]
> Odd to see this since this should be part of the every release test

We do run integration tests in every release, I have already run integration tests for `release-1.11` in both repos and they worked fine.

I tested them again on fresh deploys; they pass for both the current `release-1.11` branch and `master` (#1867) 

```
13:59:29	WARN	[test:integration]	Integration tests will erase all Panther data in account 101802775469 (us-west-2)
Are you sure you want to continue? (yes|no) yes
ok  	github.com/panther-labs/panther/cmd/opstools/requeue	22.351s
ok  	github.com/panther-labs/panther/cmd/opstools/s3queue	1.369s
ok  	github.com/panther-labs/panther/internal/compliance/compliance_api/main	3.416s
ok  	github.com/panther-labs/panther/internal/compliance/resources_api/main	2.757s
ok  	github.com/panther-labs/panther/internal/core/analysis_api/main	8.513s
ok  	github.com/panther-labs/panther/internal/core/layer_manager/main	32.809s
ok  	github.com/panther-labs/panther/internal/core/metrics_api/main	61.813s
ok  	github.com/panther-labs/panther/internal/core/organization_api/main	1.000s
ok  	github.com/panther-labs/panther/internal/core/outputs_api/main	2.086s
ok  	github.com/panther-labs/panther/internal/core/source_api/main	0.749s
ok  	github.com/panther-labs/panther/internal/core/users_api/main	3.545s
ok  	github.com/panther-labs/panther/internal/log_analysis/awsglue	3.488s
ok  	github.com/panther-labs/panther/internal/log_analysis/datacatalog_updater/process	0.670s
ok  	github.com/panther-labs/panther/pkg/awsathena	89.187s
14:04:23	INFO	[test:integration]	test:integration: python policy engine
.
----------------------------------------------------------------------
Ran 1 test in 1.095s

OK
```

----

If there is a problem with the tests, it is with

> github.com/panther-labs/panther/internal/log_analysis/datacatalog_updater/process

That's the next test that doesn't show up in your screenshot. Since your team owns the `datacatalog_updater`, I'm reassigning to you. You can test just that package by running

```
PKG=./internal/log_analysis/datacatalog_updater/process mage test:integration
```

**Comment by rleighton** [comment]
Trying that now ... btw, this is OSS repo not enterprise.

**Comment by rleighton** [comment]
Figured it out ... and it is now happening in enterprise too. The SDK must have changed. What is happening is we have added custom retriers to some of the lambdas  and we wrap the initial session with this. On a dev laptop, when getting the session it is basically hanging getting the creds because it is doing exponential backoff failing on 'EC2MetadataError: failed to make EC2Metadata request'.

I have a fix, I just need to go thru all of places we have retriers.
